test/integration/boards_test.rb

```diff
@@ -32,7 +32,7 @@ class BoardsTest < ActionDispatch::IntegrationTest
     create_card("My Card")
 
     # Then it should appear on the first list
-    within lists.first do
+    within first_list do
```

Comment:
I am fine with not having a `lists` array at all, since we agreed on having a fixed set of list. However, at this point, I'd simply remove any position related helper, and use the named ones instead. e.g.

* Use `todo_list` instead of `first_list`;
* Use `doing_list` instead of `second_list`;
* and so on...